ldi-core/ldes-client-v2/request-executor/src/main/java/ldes/client/requestexecutor/executor/RequestExecutorFactory.java

```diff
@@ -7,7 +7,7 @@
 import java.io.IOException;
 import java.time.Duration;
 
-// TODO: 6/03/2023 refactor in retry story
+// TODO retry: 6/03/2023 refactor in retry story
```

Comment:
Might be a really stupid point, but maybe add the ticket number here. Since "retry story" might become a vague term that we forget about :)

Reply:
Agree, I will add the jira ticket